process.go

```diff
@@ -106,7 +92,15 @@ func (v *Venom) parse() error {
 			}
 		}
 		if !varExtracted {
-			reallyMissingVars = append(reallyMissingVars, k)
+			var ignored bool
+			for _, i := range v.IgnoreVariables {
```

Comment:
where venom put something inside IgnoreVariables ?

Reply:
Ok, in plugin-venom only.

Reply:
yep, but if you want to add flags on command line, i can do it. 
I had a global flag to avoid parse instead